pom.xml

```diff
@@ -19,7 +19,7 @@
     <webappDirectory>
       ${project.build.directory}/${project.build.finalName}
     </webappDirectory>
-    <synapse.version>462.0</synapse.version>
+    <synapse.version>475.0</synapse.version>
```

Comment:
Thank you for updating the synapse version as well!

Reply:
It was necessary to properly serialize the JSON Subcolumns!